lib/board.rb

```diff
@@ -3,27 +3,29 @@ class Board
 
   attr_reader :cells
   def initialize
-    @cells = {
-      "A1" => Cell.new("A1"),
-      "A2" => Cell.new("A2"),
-      "A3" => Cell.new("A3"),
-      "A4" => Cell.new("A4"),
-      "B1" => Cell.new("B1"),
-      "B2" => Cell.new("B2"),
-      "B3" => Cell.new("B3"),
-      "B4" => Cell.new("B4"),
-      "C1" => Cell.new("C1"),
-      "C2" => Cell.new("C2"),
-      "C3" => Cell.new("C3"),
-      "C4" => Cell.new("C4"),
-      "D1" => Cell.new("D1"),
-      "D2" => Cell.new("D2"),
-      "D3" => Cell.new("D3"),
-      "D4" => Cell.new("D4")
-    }
+    @cells = {}
      # require "pry"; binding.pry
   end
 
+  def set_board_size
+    puts "What size would you like the board to be? Enter an integer between 1 and 26:"
+
+    size = gets.chomp.to_i
+
+    width_num = (1..size).to_a
+    height_num = width_num.map(&:clone)
+    width = width_num.map {|num| num.to_s}
+    height = height_num.map {|num| (num += 64).chr}
+
+    list = height.map do |letter|
+      width.map {|num| [letter, num].join}
+    end
+
+    list.flatten.each do |coordinate|
+      @cells[coordinate] = Cell.new(coordinate)
+    end
```

Comment:
Did you 'pry' in here to confirm the cells are generated? What happens if they input 0?